.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* stanislav.khlud@saritasa.com sergey.shirokov@saritasa.com leonid.malin@saritasa.com
```

Comment:
Maybe add more reviewers since it is public?
As I know, @OttoAndrey worked on FastAPI project now

Reply:
It' not finalized version, until public release to pipy I'll keep it as it is.